src/main/java/de/sakpaas/backend/model/Occupancy.java

```diff
@@ -10,12 +12,11 @@
 import javax.persistence.JoinColumn;
 import javax.persistence.ManyToOne;
 import javax.persistence.Table;
-import lombok.Getter;
+import javax.validation.constraints.NotNull;
+import lombok.Data;
 import lombok.NoArgsConstructor;
-import lombok.Setter;
 
-@Getter
-@Setter
+@Data
```

Comment:
Das ganze hatte @Fabsi23 erst gefixt. Es wäre glaube ziemlich Sinnlos, das ganze wieder rückgängig zu machen^^

Reply:
Zur Erklärung: @Data hat ja z.B. auch einen All-Args-Constructor, der aber definitiv vermieden werden sollte, da die ID ja Autogenerated ist

Reply:
Ich schaue nochmal nach, aber ich brauchte glaube ich die `.equals()` und somit auch die `.hashCode()` Methode, deswegen habe ich anstatt alle Annotations einzelnt hinzuzufügen einfach `@Data` verwendet :)

Reply:
Die hashCode Methode von Lombok ist ganz Böse in Models. Die kann nämlich bei Verknüpfungen (OneToMany usw.) zu Endlosschleifen führen^^

Reply:
Die kann man sich ja auch relativ easy generieren lassen

Reply:
Echt? Sehr interessant, dachte eher dass es unschön ist die `hashCode()` Methode selbst zu implementieren :) Dann kann ich das nochmal anpassen wenn du meinst, dass das schöner ist?

Reply:
@benfu99 Wie müsste ich `die hashCode()` Methode dann implementieren dass das Problem nicht auftritt? Müsste ich dann das `@OneToMany` weg lassen?

Reply:
Du dürftest halt nicht den Hash des kompletten Verknüpften Models verwenden

Reply:
Also quasi nur die `Id` wenn es eine gibt?

Reply:
genau

Reply:
Ah okay danke, dann werde ich das morgen mal implementieren.

Reply:
@benfu99 Hab ich umgesetzt :)